pkg/provisioning/keypair.go

```diff
@@ -0,0 +1,133 @@
+package provisioning
+
+import (
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"encoding/pem"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+	"golang.org/x/crypto/ssh"
+)
+
+const (
+	privateKeyFile = "serviceaccount-signer.private"
+	publicKeyFile  = "serviceaccount-signer.public"
+)
```

Comment:
Do we also want to give an option to users to name these keys?

Reply:
I think it just makes it more complicated when you run 'tool create cloud-pieces'. The step after creating keys will not be able to assume file names (unless we store some state file that says "i saved the key files as...), so you'll have to pass flags for the "happy path" (where you trust the tool to do everything).

If you want to have custom names, we will cover that will the bring-your-own-keys flag (something like 'tool create cloud-pieces --use-private-key=/path/to/private/key/filename').

WDYT?

Reply:
For now lets keep it simple and ignore this option. May be a good idea to put the names in the document so that user knows what to expect.